astropy/cosmology/io/mapping.py

```diff
@@ -17,7 +17,30 @@
 __all__ = []  # nothing is publicly scoped
 
 
-def from_mapping(map, *, move_to_meta=False, cosmology=None):
+def _rename_map(map, /, renames):
+    # So we are guaranteed to have a poppable map.
```

Comment:
What's this comment about?

Reply:
That the output guarantees a dict that is decoupled from the input map.